### PR TITLE
Settings: Change styling of Learn More links

### PR DESCRIPTION
**Pull request by MichaelArestad**
Changes styling from a link to a compact borderless button with a gridicon.

Before:
image

After:
image

### CODE REVIEW

**Comment by jeffgolenski** [comment]
omg yesssss

**Comment by jeffgolenski** [comment]
sidenote: Have you experimented with the icon location at all? I don't see it anywhere on wpcom, but moving the ? icon away from the clutter of the settings on the left, and placing it on the right may bring more visual balance? Thoughts?

**Comment by MichaelArestad** [comment]
@jeffgolenski It's only used on the help tab in Calypso. There are no help links to individual modules or settings. I was also learning toward simply removing the links altogether.

**Comment by jeffgolenski** [comment]
>  I was also leaning toward simply removing the links altogether.

As much as I love simplicity, we definitely need those in there. Some people aren't going to know how to use some features and providing direct links to the support pages will be crucial.

**Comment by MichaelArestad** [comment]
> As much as I love simplicity, we definitely need those in there. Some people aren't going to know how to use some features and providing direct links to the support pages will be crucial.

I was think that, too, but we don't have help links for some of the same stuff in Calypso and it seems to do okay.

File: _inc/client/components/module-settings/style.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
+.jp-module-settings__read-more {
+	margin-top: 16px;
```
Comment (eliorivero) [inline _inc/client/components/module-settings/style.scss]: Does this need to be passed through the `rem()` function?

**Comment by jeffgolenski** [comment]
> seems to do okay.

yeaaaa. but I don't think it hurts anything to have them? I think we should ship with them and track to see if they're actually used, and if not, we can toss them. I think they'll be important at least in this first release.

File: _inc/client/components/module-settings/style.scss (hunk 1 above)
Comment (MichaelArestad) [inline _inc/client/components/module-settings/style.scss]: Yup. Will change.

**Comment by MichaelArestad** [comment]
@jeffgolenski Fair enough. I'll fix the rem issue and merge.

**Comment by jeffgolenski** [comment]
thanks dude!

**Comment by jeffgolenski** [comment]
@MichaelArestad Can you double check this branch? Getting some npm level gulp error that only occurs within this branch, for me. 

screen shot 2016-07-30 at 3 50 56 pm

**Comment by MichaelArestad** [comment]
@jeffgolenski Confirmed, but it has to do with `npm run build` and not `gulp watch` tasks. The latter works just fine. I don't think it really has anything to do with this branch other than something node is behind master maybe. @dereksmart @oskosk @eliorivero Thoughts?

**Comment by oskosk** [comment]
checking...

**Comment by oskosk** [comment]
@MichaelArestad @jeffgolenski I could reproduce the error. Looks like the fix for this error is already in master . So if this branch gets rebased, you'll be able to build normally

```
$ git checkout update/learn-more-link
$ git rebase master
$ npm run buil
```

**Comment by oskosk** [comment]
I can do this and then review it.

Also, if it's ok to go design-wise I can merge it too after reviewing. Just tell me so :)

**Comment by eliorivero** [comment]
I'm going to rebase this one since it's needed after the introduction of the link to Email Followers in https://github.com/Automattic/jetpack/pull/4608

**Comment by eliorivero** [comment]
I'm working on this PR and will merge once it's ready.

**Comment by eliorivero** [comment]
In #4608, I added a text link so users could go to Calypso and see their follower list. Since this PR changed the Learn More text link I went ahead and changed the View your Email Followers to an icon.

#### Before, text links

captura de pantalla 2016-07-29 a las 18 21 26

#### After, icons

email-followers

#### Others
I'm not 100% sold on the icon, since I find it a bit obscure, specially in mobile, so I tried with the text link:
icons1
icons2

plus a last one that I like, maybe more than the icon solution
text

**Comment by eliorivero** [comment]
After a talk with @jeffgolenski I've implemented the alternative

text

**Comment by dereksmart** [comment]
LGTM

**Pull request merged**